editor/draw.js

```diff
@@ -31,6 +33,21 @@ var RandomizeModes = {
 var randomize_ids = RandomizeModes.LET_DOCUMENT_DECIDE;
 
 /**
+ * Add class 'layer' to the element
+ *
+ * Parameters:
+ * @param {SVGGElement} elem - The SVG element to update
+ */
+function addLayerClass(elem) {
+	var classes = elem.getAttribute('class')
+	if (classes === null || classes === undefined || classes.length === 0) {
+		elem.setAttribute('class', LAYER_CLASS);
+	} else if (! LAYER_CLASS_REGEX.test(classes)) {
+		elem.setAttribute('class', classes + ' ' + LAYER_CLASS);
+	}
```

Comment:
What do you think about using classList to do this:

    elem.classList.add(LAYER_CLASS)

http://caniuse.com/#search=classlist says support started in IE10+